composer.json

```diff
@@ -45,7 +45,8 @@
         "mockery/mockery": "^1.3.1",
         "nunomaduro/collision": "^7.0",
         "phpunit/phpunit": "^10.0",
-        "psalm/plugin-laravel": "^2.0"
+        "psalm/plugin-laravel": "^2.0",
+        "staabm/annotate-pull-request-from-checkstyle": "^1.8"
```

Comment:
I think you don't need to add the dependency here because it is already installed in the `setup-php` action.